augur/config.py

```diff
@@ -112,6 +112,40 @@
                     ],
                     "model": "releases",
                     "repo_group_id": 0
+                },
```

Comment:
These are going to be needed when we deploy the ML workers, and I would rather have them included in new installations so people don't need to update their configs when we release these workers.

Reply:
It does if you are the person maintaining 50 instances of Augur and don't want to have to manually edit the config when those are released 50 times. :)